modules/ROOT/pages/multi-factor-authentication.adoc

```diff
@@ -0,0 +1,141 @@
+= Multi-factor Authentication
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Multi-factor authentication provides an additional layer of verification for individual Anypoint Platform users.
+You can require all users in your organization to use multi-factor authentication and exempt specific accounts, or users can manually enable multi-factor authentication in their profile settings.
+
+Users who use single sign-on (SSO) cannot configure multi-factor authentication for their accounts. Instead, SSO users should configure multi-factor authentication through their identity provider.
+
+When multi-factor authentication is enabled, users must verify their Anypoint Platform credentials using at least one of the following authentication methods:
+
+* One-time password generator
+* Built-in authenticator
+* Security key
+* Salesforce Authenticator
+
+After a user has successfully verified their identity on a trusted device using multi-factor authentication, they are not required to reverify their identity for 90 days.
+
+== Multi-factor Authentication Configurations
+
+You can configure at least one of the following verification methods to comply with multi-factor authentication requirements:
+
+One-time password generator:: Registers an authenticator app to create verification codes that you provide when logging in to Anypoint Platform.
+
+Built-in authenticator:: Registers a physical authentication device, such as Touch ID, to verify your identity when logging in to Anypoint Platform.
+
+Security key:: Registers a USB security key to your account. The device is then authorized to create verification codes that you provide when logging in to Anypoint Platform.
+
+Salesforce Authenticator:: Registers the Salesforce Authenticator mobile app to create verification codes that you provide when logging in to Anypoint Platform.
+
+== Enable Multi-factor Authentication for All User Accounts
+
+For enhanced security, you can require all users in your organization to use multi-factor authentication. Users must configure their authentication method when they log in to Anypoint Platform after you initially enable multi-factor authentication.
+
+[NOTE]
+--
+To prevent service disruptions, identify any accounts that are used to programmatically call Anypoint Platform, or _service accounts_. You must add these accounts to the *Exempt Accounts* section when configuring multi-factor authentication requirements.
```

Comment:
```suggestion
To prevent service disruptions, identify any accounts that are used to programmatically call Anypoint Platform, or _service accounts_. You must either configure these calls using the xref:connected-apps-overview.adoc[Connected Apps] feature or add these accounts to the *Exempt Accounts* section when configuring multi-factor authentication requirements. 
```